.github/workflows/CI.yml

```diff
@@ -21,6 +21,12 @@ jobs:
 
       - name: Setup .NET
         run: apt install -y dotnet6
+      - name: Set execute permissions for dummy files
+        run: |
+          chmod +x src/FileConventions.Test/DummyFiles/DummyExecutable.fsx
+          chmod -x src/FileConventions.Test/DummyFiles/DummyNotExecutable.fs
+          ls -l src/FileConventions.Test/DummyFiles/DummyExecutable.fsx
+          ls -l src/FileConventions.Test/DummyFiles/DummyNotExecutable.fs
```

Comment:
@realmarv these ls calls are for debugging, I guess, so they should be removed

Reply:
Done